engines/glk/adrift/os_glk.cpp

```diff
@@ -2811,7 +2811,7 @@ static int gsc_startup_code(Common::SeekableReadStream *game_stream, int restore
 		 * Display a brief loading game message; here we have to use a timeout
 		 * to ensure that the text is flushed to Glk.
 		 */
-		g_vm->glk_put_string(_("Loading game...\n"));
+		g_vm->glk_put_string("Loading game...\n");
```

Comment:
This code doesn't even work in current master (non-ASCII characters are not displayed properly). However the GLK engine actually handles unicode text, and there is a `glk_put_string_uni(const uint32 *)` function for example. So with the change to use u32 strings for the translation, we could actually make this code work I think.